### DISCUSSION TITLE
Ory Kratos Python SDK example

### DISCUSSION THREAD

**Discussion by nihalgurjar47** [ory/awesome-ory#80] [Q&A]
Hi @vinckr ,

I want a user registration and user login flow using the Kratos Python SDK. If you have any existing example repositories, please share them.

**Comment by vinckr** [comment]
Hey @nihalgurjar47 

Here is an example for Django: https://github.com/ory/examples/tree/master/django-ory-cloud

Would be great to have an "official" example in the documentation though!

**Reply by nihalgurjar47** [reply]
Thanks for the update @vinckr 

But my requirement is self-hosting, so if you have any examples related to self-hosting, please share them.

**Reply by vinckr** [reply]
You can use the same example for self-hosted Ory as well @nihalgurjar47 - just switch out the endpoints. 

Have a look at community resources as well: https://www.ory.sh/docs/ecosystem/community

Feel free to contribute any Python example in the ory/examples repository as well!
